index.bs

```diff
@@ -2757,6 +2758,29 @@ Note: Invoking this method from a [=browsing context=] where the [=Web Authentic
 
 </div>
 
+### Availability of a [=passkey platform authenticator=] - PublicKeyCredential's `isPasskeyPlatformAuthenticatorAvailable()` Method ### {#sctn-isPasskeyPlatformAuthenticatorAvailable}
+
+<div link-for-hint="WebAuthentication/isPasskeyPlatformAuthenticatorAvailable">
+
+[=[WRPS]=] use this method to determine whether they can create a new [=passkey=] using a [=user-verifying platform authenticator=] or a {{AuthenticatorTransport/hybrid}} authenticator.
+Upon invocation, the [=client=] employs a [=client platform=]-specific procedure to discover available [=user-verifying platform authenticators=] and the 
+availability of {{AuthenticatorTransport/hybrid}} transport.
+If one or both are discovered, the promise is resolved with the value of [TRUE].
+If neither are discovered, the promise is resolved with the value of [FALSE].
```

Comment:
I think?
```suggestion
If neither is discovered, the promise is resolved with the value of [FALSE].
```

Reply:
Need a third person to do a grammar check. "Is" doesn't sound correct to me. @MasterKale?

Reply:
So the **English Language & Usage** Stack Exchange claims that "neither is" is _technically_ correct, but "neither are" is more common. Do specs like ours aim for technical correctness is all aspects, or are colloquialisms allowed in?

Personally I say "neither are" because it sounds more natural, but if it's for sake of "linguistic clarity" across the board I'd say go with "neither is".

https://english.stackexchange.com/questions/10304/which-is-correct-neither-is-or-neither-are